src/overlays/actors/ovl_Door_Warp1/z_door_warp1.h

```diff
@@ -7,14 +7,52 @@ struct DoorWarp1;
 
 typedef void (*DoorWarp1ActionFunc)(struct DoorWarp1*, GlobalContext*);
 
+#define DOORWARP1_GET_FF(thisx) ((thisx)->params & 0xFF)
+#define DOORWARP1_GET_FF00_1(thisx) (((thisx)->params >> 8) & 0xFF)
+#define DOORWARP1_GET_FF00_2(thisx) ((thisx)->params & 0xFF00)
+#define DOORWARP1_GET_FF00_3(thisx) ((thisx)->params >> 8)
+
+enum {
+    /* 0 */ ENDOORWARP1_FF_0,
+    /* 1 */ ENDOORWARP1_FF_1,
+    /* 2 */ ENDOORWARP1_FF_2,
+    /* 3 */ ENDOORWARP1_FF_3,
+    /* 4 */ ENDOORWARP1_FF_4,
+    /* 5 */ ENDOORWARP1_FF_5,
+    /* 6 */ ENDOORWARP1_FF_6,
+};
+
 typedef struct DoorWarp1 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x94];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ SkelAnime skelAnime;
+    /* 0x01A0 */ struct DmHina* unk_1A0;
```

Comment:
You can remove the `struct` keyword here since that struct is being typedef'd in its corresponding header.
```suggestion
    /* 0x01A0 */ DmHina* unk_1A0;
```

Btw, you should `#include` that header in this header explicitly

Reply:
I usually put it in the c file, want me to move it into the .h instead?